test/e2e/os/arch.go

```diff
@@ -58,7 +59,7 @@ func getAmiIDFromSSM(ctx context.Context, client *ssm.SSM, amiName string) (*str
 }
 
 func getInstanceTypeFromRegionAndArch(region, arch string) string {
-	if arch == amd64Arch {
+	if arch == amd64Arch || arch == x8664Arch {
```

Comment:
can we just normalize the arch string before passing it to this method?

Reply:
But that's just going to be another method with a similar control statement just the arch. I didn't do that to avoid the duplication.

Reply:
Why is another method bad? 

I like the idea bc it would encapsulate the logic to determine the architecture in each os, making this code decoupled from the per os string values. It also allows you to describe some of the business logic in the type system by using an "enum".

Reply:
Not bad per se, just went with what IMO was the easiest solution for the short term, but I have updated it to pass in the normalized arch.